.github/workflows/golangci-lint.yaml

```diff
@@ -0,0 +1,24 @@
+name: golangci-lint
+on: [push, pull_request]
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.14
+      - name: Print go version
+        run: go version
+      - name: Check out module
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 1
+
+      - name: Install golangci-lint
```

Comment:
Hi,
did you look at [this action](https://github.com/golangci/golangci-lint-action)?

Reply:
d'oh! 
Thank you for spotting this. I was looking everywhere, but not at the actual source itself.
I'll change this to the official the next time I come around :)